src/containers/find-offer/filter-bar-menu/FilterBarMenu.constants.ts

```diff
@@ -0,0 +1,38 @@
+import { CardsViewEnums } from '~/types/findOffers/enums/viewSwitcher.enums'
+
+
+export const switcherOptions = {

```

Comment:
is it needed ? you are using the same object in component

Reply:
as for me it was looking cleaner, but ok, i have refactored